spec/factories.rb

```diff
@@ -146,6 +146,29 @@
     initialize_with { Provider.find_or_create_by code: code }
     code { Faker::Alphanumeric.alphanumeric(number: 3).upcase }
     name { Faker::Educator.university }
+
+    transient do
+      provider_agreements_count { 1 }
```

Comment:
Do we need to default to having an agreement here? I think this will generate a lot more data agreements than necessary.

Reply:
Yes, all the other system specs break otherwise (no agreement = going through agreement flow)